test/metabase/email_test.clj

```diff
@@ -113,18 +113,18 @@
   (regex-email-bodies* regexes @inbox))
 
 (defn received-email-subject?
-  "Indicate whether user the user received an email whose subject matches the `regex`. User should be a keyword
-  like :rasta."
-  [user regex]
-  (let [address (:username (user/user->credentials user))
+  "Indicate whether a user received an email whose subject matches the `regex`. First argument should be a keyword
+  like :rasta, or an email address string."
+  [user-or-email regex]
+  (let [address (if (string? user-or-email) user-or-email (:username (user/user->credentials user-or-email)))
         emails  (get @inbox address)]
     (boolean (some #(re-find regex %) (map :subject emails)))))
 
 (defn received-email-body?
-  "Indicate whether user the user received an email whose body matches the `regex`. User should be a keyword
-  like :rasta."
-  [user regex]
-  (let [address (:username (user/user->credentials user))
```

Comment:
@dpsutton FYI, changing these helpers to accept either a user keyword or an email string as the first arg (since the latter is what is used in the setup test). LMK what you think.

Reply:
that sounds like a great addition.

Reply:
agreed. awesome 👍